recipes/newrelic/infrastructure/ohi/kafka/debian.yml

```diff
@@ -0,0 +1,53 @@
+# Visit our schema definition for additional information on this file format
+# https://github.com/newrelic/open-install-library/blob/main/docs/recipe-spec/recipe-spec.md#schema-definition
+
+name: kafka-open-source-integration
+displayName: kafka Open Source Integration
+description: New Relic install recipe for default kafka Open Source on-host integration (via Infra-Agent)
+repository: https://github.com/newrelic/nri-kafka
+
+dependencies:
+  - infrastructure-agent-installer
+
+installTargets:
+  - type: host
+    os: linux
+    platform: "debian"
+  - type: host
+    os: linux
+    platform: "ubuntu"
+
+keywords:
+  - Infrastructure
+  - Integration
+  - kafka
+
+processMatch:
+  - kafka
+
+preInstall:
+  info: |2
+      To capture data from the kafka integration, you'll first need to meet these prerequisites:
+      - kafka version requirement (see https://docs.newrelic.com/docs/infrastructure/host-integrations/host-integrations-list/kafka-monitoring-integration/#req)
+  requireAtDiscovery: |
+    if ss -tupln | grep -q 9092; then exit 132; else exit 1; fi
+
+install:
+  version: "3"
+  silent: true
+
+  tasks:
+    default:
+      cmds:
+        - task: setup
+
+    setup:
+      cmds:
+        - |
+          exit 131
+
+postInstall:
+  info: |2
+      ⚙️  The kafka configuration file can be found in /etc/newrelic-infra/integrations.d/kafka-config.yml
+      Edit this file to make changes or configure advanced features for this integration. See the docs for options:
+      https://docs.newrelic.com/docs/infrastructure/host-integrations/host-integrations-list/kafka-monitoring-integration/#configg
```

Comment:
@NRhzhao there is a typo on the url, the last config word has 2 'g'

Reply:
@Julien4218  Sure, I wasn't sure because other recipes all have them separated.  I figure it's due to they have different package managers, so during installation, we might keep them seperate.